README.md

```diff
@@ -1 +1,13 @@
 # Bhukkad
+Food overflow app! :)
+
+# Contributions
+
+## Branch naming convention
+feature/desc - Implementing a new feature, refactoring, optimisation
+bugfix/desc - Any bug fixes
+
+## Code commit convention
+feat: desc - Implementing a new feature
```

Comment:
lets make the commit message convention like this 
[Feature] Desc
[Fix] Desc
[Refactor] Desc